students/turner.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <title>Methods 2022 - Students - Turner Burchard</title>
+</head>
+<body>
+    <h1>Turner Burchard</h1>
+
+    <p>
+	I'm in my second year of Computer Science (Professional Option) at MSU. I'm currently 
+	working for Brightvine as an intern in data engineering. Some of my favorite hobbies 
+	include skiing, hiking, mountain biking, climbing, drawing, gaming, and cooking.
+    </p>
+    <ul>
+        <li><a href="https://github.com/turnerburchard">Github</a></li>
+        <li><a href="https://www.linkedin.com/in/turner-burchard">LinkedIn </a></li>
+        <li><a href="https://turnerburchard.com/resume">Resume</a></li>
+    </ul>
+    <p></p>
```

Comment:
unused <p> tag

Reply:
not very good at web design... easy way to add vertical whitespace